### DISCUSSION TITLE
Пустые файлы

### DISCUSSION THREAD

**Discussion by rotaliham** [lirix360/ReadmangaGrabber#73] [Q&A] [answered]
Начинаю скачивать мангу с https://mintmanga.live/chelovek_benzopila__A5327 , вроде качает, да создает пустые папки.

**Comment by lirix360** [comment, marked as answer]
> Начинаю скачивать мангу с https://mintmanga.live/chelovek_benzopila__A5327 , вроде качает, да создает пустые папки.

Большинство манги на МинтМанга требует входа на сайт для просмотра, как это сделать в утилите [написано тут](https://github.com/lirix360/ReadmangaGrabber#%D1%81%D0%BE%D1%85%D1%80%D0%B0%D0%BD%D0%B5%D0%BD%D0%B8%D0%B5-cookies-%D0%B4%D0%BB%D1%8F-readmangamintmanga).

**Reply by rotaliham** [reply]
Все сделал по инструкции, не идет

**Reply by lirix360** [reply]
> Все сделал по инструкции, не идет

Исправлено в [версии 2.1.5](https://github.com/lirix360/ReadmangaGrabber/releases/tag/v2.1.5)

**Comment by TheHiddenSchemer** [comment]
Такая же проблема, но не на минтманга, + зареган.
И блин прога только заработала, а на 3 тайтл всё перестала
